x-pack/plugins/index_lifecycle_management/server/routes/api/index/register_add_policy_route.ts

```diff
@@ -9,6 +9,7 @@ import { ElasticsearchClient } from 'kibana/server';
 
 import { RouteDependencies } from '../../../types';
 import { addBasePath } from '../../../services';
+import { handleEsError } from '../../../shared_imports';
```

Comment:
nit: Instead of importing `handleEsError` everywhere, we could do something similar to what we were doing previously with `isEsError` and only import it in the main `server/plugin.ts` file as a parameter to `registerApiRoutes`.

https://github.com/elastic/kibana/blob/7.10/x-pack/plugins/index_lifecycle_management/server/plugin.ts#L102